www-misc/libkiwix/libkiwix-13.1.0.recipe

```diff
@@ -24,15 +24,13 @@ PROVIDES="
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libcurl$secondaryArchSuffix
-	lib:libgnutls$secondaryArchSuffix
 	lib:libicui18n$secondaryArchSuffix
-	lib:liblzma$secondaryArchSuffix
+	lib:libicuuc$secondaryArchSuffix
 	lib:libmicrohttpd$secondaryArchSuffix
 	lib:libpugixml$secondaryArchSuffix
 	lib:libxapian$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
 	lib:libzim$secondaryArchSuffix
-	lib:libzstd$secondaryArchSuffix
```

Comment:
Not going to hold back the merge on this, but this change also reflects the issue at: https://github.com/haikuports/haikuports/issues/11268

libmicrohttpd can't be found without libgnutls
libzim can't be found without liblzma and libzstd

The tree libraries itself are not used or linked by in libkiwix